modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFRequest.testcase

```diff
@@ -225,6 +225,74 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-174679.Verify that is possible to export filtered MDF’s"
+	@priority = 4
+	test CanExportFilteredMDFs {
+		task ("Given a new account is created") {
+			JSONAccountEntry.addAccountEntry(
+				accountEntryName = "Company Name 2",
+				accountEntryType = "Business");
+
+			JSONAccountEntryUser.addExistUsertoAccount(
+				accountEntryName = "Company Name 2",
+				userEmailAddress = "partner@liferay.com");
+		}
+
+		task ("And two MDF's are created with different accounts") {
+			JSONPRM.createMDFRequest(
+				accountEntryName = "Company Name 1",
+				campaignName = "Campaign Name 1");
+
+			JSONPRM.createMDFRequest(
+				accountEntryName = "Company Name 2",
+				campaignName = "Campaign Name 2");
+		}
+
+		task ("And I go to MDF Listing Page") {
+			PRMUtils.gotoPRM();
+
+			PRMHomePage.gotoQuickLinksOption(
+				buttonLink = "mdf",
+				buttonText = "MDF");
+		}
+
+		task ("When I filter the MDF by Account and selects 'Company Name 1") {
+			PRMMDFRequest.viewFilterMDF(
+				boxName = "Company Name 1",
+				keyFilter = "Filter",
+				keyPartner = "Partner");
+		}
+
+		task ("When I click on the 'Export MDF Report' button") {
```

Comment:
Hi @juliosilveira77, this task should be 'And', instead of 'When'. Could you please replace it? Thanks!